iwyu.cc

```diff
@@ -1638,7 +1641,7 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
   // The comment, if not nullptr, is extra text that is included along with
   // the warning message that iwyu emits. The extra use flags is optional
   // info that can be assigned to the use (see the UF_* constants)
-  virtual void ReportDeclUse(SourceLocation used_loc,
+  virtual void ReportDeclUse(const ASTNode& used_node,
```

Comment:
Is it `used_node` or `use_node`? I think it denotes _where_ `used_decl` is actually used, so `use_node` seems better... Not sure. :man_shrugging: